ext/xmlrpc/xmlrpc.stub.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+function xmlrpc_encode($value): string {}
+
+function xmlrpc_decode(string $xml, string $encoding = "iso-8859-1") {}
+
+function xmlrpc_decode_request(string $xml, &$method, string $encoding = "iso-8859-1") {}
+
+function xmlrpc_encode_request(string $method, $params, array $output_options = UNKNOWN): string {}
+
+function xmlrpc_set_type(&$value, string $type): bool {}
+
+function xmlrpc_is_fault(array $arg): bool {}
+
+/** @return resource */
+function xmlrpc_server_create() {}
+
+/** @param resource $server */
+function xmlrpc_server_destroy($server): bool {}
+
+/** @param resource $server */
+function xmlrpc_server_register_method($server, string $method_name, $function): bool {}
+
+/**
+ * @param resource $server
+ * 
+ * @return string|false
+ */
+function xmlrpc_server_call_method($server, string $xml, $user_data, $output_options = UNKNOWN) {}
```

Comment:
`$output_options` has to be an `array`, and it seems the function can return any PHP type, so:
```suggestion
 * @return mixed
 */
function xmlrpc_server_call_method($server, string $xml, $user_data, array $output_options = UNKNOWN) {}
```